src/compiler/emitter.ts

```diff
@@ -3,6 +3,190 @@
 
 /* @internal */
 namespace ts {
+    export const defaultInitCompilerOptions: CompilerOptions = {
+        module: ModuleKind.CommonJS,
+        target: ScriptTarget.ES3,
+        noImplicitAny: true,
+        outDir: "built",
+        rootDir: ".",
+        sourceMap: false,
+    }
+
+    export function buildConfigFile(writer: EmitTextWriter, compilerOptions: CompilerOptions, fileNames?: string[], excludes?: string[]) {
+        compilerOptions = extend(compilerOptions, defaultInitCompilerOptions);
+        let { write, writeLine, increaseIndent, decreaseIndent } = writer;
+        let { optionNameMap } = getOptionNameMap();
+        writeTsConfigDotJsonFile();
+
+        function writeTsConfigDotJsonFile() {

```

Comment:
I would prefer to have at least the comments. Because people might not know what options do what. This is more inline with VSCode they also support comments and they come as default in the debug settings explaining every option.

Trailing comma is not supported in VSCode though. I could probably remove it.